demos/2d_cylindrical/2d_cylindrical.py

```diff
@@ -38,6 +38,7 @@
 rmin, rmax, ncells, nlayers = 1.22, 2.22, 128, 32
 mesh1d = CircleManifoldMesh(ncells, radius=rmin, degree=2)  # construct a circle mesh
 mesh = ExtrudedMesh(mesh1d, layers=nlayers, extrusion_type='radial')  # extrude into a cylinder
+mesh.cartesian = False
```

Comment:
Since the explanation about cartesian is removed later in this file, I guess there should be something in the cell above about this change.

Reply:
Agreed. Perhaps update the text above to include:

# We specify that the problem is not executed in a Cartesian geometry, by setting the mesh.cartesian argument to false. This ensures that boundary conditions and specification of the vertical/radial direction are configured appropriately.